.gitignore

```diff
@@ -19,6 +19,7 @@ pip-log.txt
 # Unit test / coverage reports
 .coverage
 .tox
+.cache
```

Comment:
It was pytest, it added test results to that folder - I wasn't quite sure about that either.